### PR TITLE
Russian translation of "A Response to Reuters"

### PR DESCRIPTION
**Pull request by v1docq47**
_add:_
>  monero-outreach-docs/translations/ru/archive/reuters_response_ru.md

_translated:_
>  monero-outreach-docs/translations/ru/archive/reuters_response_ru.md

Calling to review:
@monerorus @LvMsterfild @TheFuzzStone

### CODE REVIEW

**Comment by TheFuzzStone** [comment]
Done.

File: monero-outreach-docs/translations/ru/archive/reuters_response_ru.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Наш ответ Reuters
+*05/20/2019*
+Reuters опубликовали статью, посвящённую Monero, и нам хотелось бы поделиться своей точкой зрения.
+
+На прошлой неделе, 14 мая 2019, Reuters опубликовали статью, в которой были описаны особенности и характеристики криптовалюты Monero ([reuters.com/article/us-crypto-currencies-altcoins-explainer-idUSKCN1SL0F0](https://www.reuters.com/article/us-crypto-currencies-altcoins-explainer-idUSKCN1SL0F0)). ы хотели бы поблагодарить новостное агентство за серию статей, посвящённых альтернативам Bitcoin, а также за выбор Monero в числе первых!
```
Comment (monerorus) [inline monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: **М**ы хотели бы поблагодарить

File: monero-outreach-docs/translations/ru/archive/reuters_response_ru.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Bitcoin полностью прозрачен. Любой тинейджер, у которого есть подключение к сети интернет, сможет узнать о вас больше (если вы пользуетесь Bitcoin), чем ему позволил закон, если вы пользовались бы фиатной валютой. Monero не прозрачна. *ледовательно, Monero не является «анонимной монетой», а представляет собой «взаимозаменяемую монету», которая просто реплицирует взаимную анонимность, которой мы обладаем в рамках нашей законной финансовой системы, системы, которая обеспечивает свободу в обществе*. Bitcoin и другие прозрачные монеты являются огромным шагом назад от того, чем нас наделило общество. И Monero была создана как решение этой проблемы.
```
Comment (monerorus) [inline monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: > *ледовательно, Monero не является «анонимной монетой»

File: monero-outreach-docs/translations/ru/archive/reuters_response_ru.md (hunk 1 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: fix

File: monero-outreach-docs/translations/ru/archive/reuters_response_ru.md (hunk 2 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: fix

**Comment by v1docq47** [comment]
@TheFuzzStone @LvMsterfild @monerorus thanks for review

File: monero-outreach-docs/translations/ru/archive/reuters_response_ru.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Наш ответ Reuters
+*05/20/2019*
+Reuters опубликовали статью, посвящённую Monero, и нам хотелось бы поделиться своей точкой зрения.
```
Comment (erciccione) [inline monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: Please add 2 trailing spaces at the end of the line or the format will result broken. Same for the string above

Comment (v1docq47) [reply monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: fix ;)

Comment (erciccione) [reply monero-outreach-docs/translations/ru/archive/reuters_response_ru.md]: Sorry i wasn't clear. The 2 trailing spaces after ` # Наш ответ Reuters` were not needed, but it's fine anyway. Mergin this now

**Pull request merged**